docs/index.html

```diff
@@ -209,6 +209,42 @@ <h1>Electronics and electrical design checklist</h1>
 				</li>
 			</ul>
 		</li>
+		<li>EMC
+			<ul>
+				<li> Requirements
+					<ul>
+						<li> What noise freqencies are emitted? (e.g. clock and SMPS frequencies) </li>
+						<li> What noise frequencies is the circuit sensitive to? (RF?) </li>
+						<li> What noise will be generated by connected devices? (e.g. mains adaptors) </li>
+						<li> What countries will the product be sold in? What standards cover these countries? </li>
+						<li> What is the end use for the product?
+							<div>
+							Industrial or automotive use has different test requirements and permissable emission levels than consumer goods. For industrial, the levels are usually higher. See <a href="https://www.conformance.co.uk/adirectives/doku.php?id=emc#tests"> Conformance UK's guide </a>
+							</div> 
+						</li>
+					</ul>
+				</li>
+				<li> Prevention
+					<ul>
+						<li>Do switching supplies have (or have option for) RC snubbers?
+							<div>While not always needed for stability, an RC snubber can dampen ringing oscillations, which result in high frequency noise that may exceed EMC limits.</div>
+						</li>
+						<li> Is differential signalling used for digital IO where possible? 
+							<div> Differential signals are not only more immune to noise, but emit less as well. For high speed, long signal lines (e.g. remote displays in automotive), the difference can be considerable.
+							</div>
+						</li>
+						<li> Is differential signalling used for analog IO? </li>
+						<li> Are shielded connectors needed? </li>
+						<li> Is a shielded case needed? How will this be bonded to ground? </li>
+						<li> Are switch mode power supplies syncronized? 
+							<div> 
+							If switch mode power supplies are powered from the same source and are not syncronized, they can produce "beat" frequencies from being out of phase, and input and output ripple is also increased. See <a href="Power Tip 73: Synchronizing Makes for Well-Behaved Power Supplies"> TI Power Tip 73: Synchronizing Makes for Well-Behaved Power Supplies </a>
```

Comment:
```suggestion
							If switched-mode power supplies are powered from the same source and are not syncronized, they can produce "beat" frequencies from being out of phase, and input and output ripple is also increased. See <a href="Power Tip 73: Synchronizing Makes for Well-Behaved Power Supplies"> TI Power Tip 73: Synchronizing Makes for Well-Behaved Power Supplies </a>
```

The `href` is not the URL of the document.

Reply:
```suggestion
							If switch mode power supplies are powered from the same source and are not syncronized, they can produce "beat" frequencies from being out of phase, and input and output ripple is also increased. See <a href="https://www.eetimes.com/author.asp?section_id=183&doc_id=1323282"> TI Power Tip 73: Synchronizing Makes for Well-Behaved Power Supplies </a>
```